### PR TITLE
docs: Added minimum node version to DevelopmentNotes.rst

### LINKED ISSUE #406: Documentation: Add min node version for local setup
### What is the current documentation state?

In [Development Notes](https://github.com/moja-global/FLINT-UI/blob/master/docs/DevelopmentGuide/DevelopmentNotes.rst), the local setup for the UI doesn't mention the min. node version. 

### Where is this stated?

https://github.com/moja-global/FLINT-UI/blob/master/docs/DevelopmentGuide/DevelopmentNotes.rst

### Why do you want to improve the statement?

Faster and easier local setup

### Proposed Statement

Mention the min. node version required

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by abhilipsasahoo03**
## Description

This PR fixes #406 

## Change log

- Added minimum node version required to set up the project locally to DevelopmentNotes.rst
- Added installation guide for Node and Yarn to DevelopmentNotes.rst in order to help new contributors get started
- Fixed minor typo

### CODE REVIEW

**Comment by abhilipsasahoo03** [comment]
@chicken-biryani kindly look into this

**Review by chicken-biryani — APPROVED** [review]

File: docs/DevelopmentGuide/DevelopmentNotes.rst [hunk 1]
```diff
@@ -34,6 +34,44 @@ GCBM
 
    docker run --rm -d -p 8081:8080 gcbm-api
 
+Node
+-----
+
+This project requires node version >= 14
```
Comment (HarshCasper) [inline docs/DevelopmentGuide/DevelopmentNotes.rst:40]: `node` -> `Node`

File: docs/DevelopmentGuide/DevelopmentNotes.rst [hunk 2]
```diff
@@ -34,6 +34,44 @@ GCBM
...
+
+1. To check the version of installed node, run the following command:
+
+.. code:: shell
+
+   node -v
+   
+2. You can also replace ``-v`` with ``--version`` and it should work the same.
+
+3. To update node to the latest version, ensure nvm is installed and run the following command:
```
Comment (HarshCasper) [inline docs/DevelopmentGuide/DevelopmentNotes.rst:50]: Link `nvm` here and put it in backticks

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by abhilipsasahoo03** [comment]
@HarshCasper please take a look

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**